addon/index.ts

```diff
@@ -11,30 +12,33 @@ function resolveInitializer(moduleName) {
   return initializer;
 }
 
-function registerInitializers(app, moduleNames) {
+function registerInitializers(app: typeof Engine, moduleNames: string[]) {
   for (var i = 0; i < moduleNames.length; i++) {
     app.initializer(resolveInitializer(moduleNames[i]));
   }
 }
 
-function registerInstanceInitializers(app, moduleNames) {
+function registerInstanceInitializers(app: typeof Engine, moduleNames: string[]) {
   for (var i = 0; i < moduleNames.length; i++) {
     app.instanceInitializer(resolveInitializer(moduleNames[i]));
   }
 }
 
-function _endsWith(str, suffix) {
+function _endsWith(str: string, suffix: string) {
   return str.indexOf(suffix, str.length - suffix.length) !== -1;
 }
 
-export default function (app, prefix) {
+/**
+ * Configure your application as it boots
+ */
+export default function loadInitializers(app: typeof Engine, prefix: string) {
```

Comment:
I like explicitly stating the return value, I know it doesn't matter 😸 
```suggestion
export default function loadInitializers(app: typeof Engine, prefix: string): void {
```